modules/apps/depot/depot-test/src/testFunctional/tests/DepotMemberships.testcase

```diff
@@ -0,0 +1,142 @@
+@component-name = "portal-document-management"
+definition {
+
+	property portal.release = "false";
```

Comment:
@Jeremy-Ch Can you file a test improvement to change this to "true?" I forgot to change it